pcmdi_metrics/precip_variability/scripts_pcmdi/calc_ps_area_freq_mean_regional.py

```diff
@@ -0,0 +1,152 @@
+import copy
+import datetime
+import json
+import os
+import pickle
+
+import numpy as np
+
+
+def prdday_to_frq3hridx(prdday, frequency):
+    frq3hr = 1.0 / (float(prdday) * 8.0)
+    idx = (np.abs(frequency - frq3hr)).argmin()
+    return int(idx)
+
+
+def prdday_to_frq1didx(prdday, frequency):
+    frq24hr = 1.0 / (float(prdday))
+    idx = (np.abs(frequency - frq24hr)).argmin()
+    return int(idx)
+
+
+# --------------------
+# User settings here
+# --------------------
+# "3hr" or "day"
+hr = ""
+# Input file name (pickle .pkl output from calc_ps_area.mean.py)
+fname = ""
+# Output directory (not including version)
+outdir = ""
+
+# -----------------------
+# The rest of the script
+# -----------------------
+ver = datetime.datetime.now().strftime("v%Y%m%d")
+
+if hr == "day":
```

Comment:
@msahn I removed diurnal and semi-diurnal from the daily timescale case. Is that reasonable?

Reply:
Yes, that seems reasonable. I also use the same logic in the lib code as below.
 https://github.com/PCMDI/pcmdi_metrics/blob/035f654da98301538c8a1c693449435b346c37fb/pcmdi_metrics/precip_variability/lib/lib_variability_across_timescales.py#L368-L379